master/reference/calicoctl/resources/index.md

```diff
@@ -43,42 +43,51 @@ spec:
 
 The following resources are supported:
 
-- [bgpPeer]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/bgppeer)
-- [hostEndpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint)
-- [policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy)
-- [ipPool]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
-- [profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile)
-- [workloadEndpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
+- [BGPPeer]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/bgppeer)
+- [BGPConfiguration]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/bgpconfig)
+- [HostEndpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint)
+- [NetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy)
+- [GlobalNetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globalnetworkpolicy)
+- [IPPool]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+- [Profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile)
+- [FelixConfiguration]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/felixconfig)
+- [ClusterInformation]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/clusterinfo)
+- [WorkloadEndpoint]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
+
+### Resource name requirements
+
+Every resource must have the `name` field specified.
+A valid resource name can have alphanumeric charecters with optional `.`, `_`, `-`, or `/` of up to the length of 128 charecters total.
```

Comment:
"charecters" --> "characters" (2X)
I also think you can and should lose "the length of" so.... "up to the length of 128" --> "up to 128"